### PR TITLE
Make too many rows warning limit configurable

### PR DESCRIPTION
**Pull request by Smotko**
This should resolve https://github.com/dbcli/pgcli/issues/519

### CODE REVIEW

File: pgcli/main.py [hunk 1]
```diff
@@ -516,9 +523,8 @@ def _evaluate_command(self, text):
             logger.debug("headers: %r", headers)
             logger.debug("rows: %r", cur)
             logger.debug("status: %r", status)
-            threshold = 1000
-            if (is_select(status) and
-                    cur and cur.rowcount > threshold):
+            threshold = self.row_limit
+            if (self._should_show_limit_prompt(status, cur)):
```
Comment (drocco007) [inline pgcli/main.py]: Let's remove the extra parentheses around the if test.

**Comment by amjith** [comment]
@Smotko Thanks for the PR. It looks great. 

@drocco007 Can you review this and merge? I presume you were shepherding @Smotko.

**Comment by drocco007** [comment]
Hey @Smotko! Thanks again for your help!

When you changed how the default is specified, we lost the implicit conversion to `int` that Click does if the default is an integer, so setting the row type via `-R` or the environment variable doesn't work. Fortunately the fix should be pretty easy: just add `type=click.INT` to your option definition. Also, please rebase this your branch against the updated master. Thanks!

cc: @amjith

**Comment by Smotko** [comment]
@drocco007 ah, sorry about that. Should be fixed now.

**Comment by amjith** [comment]
Works great! Thanks for the PR @Smotko. 

🍨

**Comment by drocco007** [comment]
Thanks @Smotko! Nice working with you at PyCon! :)

**Pull request merged**